interfaces/kepware/kepwareClient.js

```diff
@@ -0,0 +1,200 @@
+// node-opcua documentation: http://node-opcua.github.io/api_doc/2.0.0/index.html
+
+/********************************** Imports ***********************************/
+
+const {
+    OPCUAClient,
+    MessageSecurityMode,
+    SecurityPolicy,
+    AttributeIds,
+    TimestampsToReturn,
+    NodeClass,
+    Variant,
+    DataType,
+    findServers
+} = require('node-opcua');
+
+/******************************* General Config *******************************/
+
+const subscriptionParameters = {
+  requestedPublishingInterval: 300, // How often server pushes updates
```

Comment:
Is this something that could be made configurable via the UI? (Am I interpreting correctly that this means it will subscribe to data coming from kepware on a 300ms delay? Some use-cases might want faster data rates)